src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: containerWidth($squareSide, $gap, $border);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $squareSide;
+    height: $squareSide;
+    border: $border solid black;
+    background-color: #eee;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    transition: 0.5s;
+  }
+
+  &__day:hover {
+    background-color: #ffc0cb;
+    transform: translateY(-20px);
+    cursor: pointer;
```

Comment:
Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.